traits/trait_types.py

```diff
@@ -1387,7 +1388,7 @@ class BaseFile(BaseStr):
     """
 
     #: A description of the type of value this trait accepts:
-    info_text = "a file name"
+    info_text = "a file name or PathLike object"
```

Comment:
Suggest "instance of os.PathLike" instead of "PathLike object", since `PathLike` doesn't get instantiated directly. "object implementing the os.PathLike interface" would work, too.